Cargo.toml

```diff
@@ -28,6 +28,7 @@ alloc = []
 arrayvec = { version = "0.7", default-features = false }
 
 core2 = { version = "0.3.2", default-features = false, optional = true }
+io = { package = "bitcoin-io", version = "0.1.1", default-features = false, optional = true }
```

Comment:
Please don't rename this.

Reply:
Sure I'll remove it, whats the reason?

Reply:
I've already explained it somewhere else - someone who's not from `bitcoin` would find the code confusing.